redis/options.hh

```diff
@@ -47,19 +47,15 @@ class redis_options {
     size_t _total_redis_db_count;
 	unsigned _db_index;
 public:
-    //redis_options(redis_options&&) = default;
-    
-    //explicit redis_options(const redis_options&) = default;
-   
     explicit redis_options(const db::consistency_level rcl,
         const db::consistency_level wcl,
         const timeout_config& tc,
         auth::service& auth,
         const socket_address addr,
         size_t total_redis_db_count)
-		: redis_options(0, rcl, wcl, tc, auth, addr, total_redis_db_count)
-	{
-	}
+    : redis_options(0, rcl, wcl, tc, auth, addr, total_redis_db_count)
```

Comment:
This cleanups have nothing to do with this patch (adding a test), but I'll let it slide :-)